uav_scanning:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  camera_pose_controller:
    type: effort_controllers/JointPositionController
    joint: camera_joint
    pid: {p: 100.0, i: 0.01, d: 2.0}
